basiclibrary/src/main/java/basiclibrary/Lab3Examples.java

```diff
@@ -0,0 +1,66 @@
+package basiclibrary;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+
+public class Lab3Examples {
+
+
+    public static String maps (int[][] weeklyMonthTemperatures){
+
+        String outputStr = "Never saw temperature: ";
+
+        //HashSet
+        HashSet<Integer> temperatures = new HashSet<>();
+
+        //StringBuilder
+        StringBuilder buildStr = new StringBuilder();
+
+
+        //Adds temperature values into HashSet
+        for (int i = 0; i < weeklyMonthTemperatures.length; i++) {
+            for (int j = 0; j < weeklyMonthTemperatures[i].length ; j++) {
+                temperatures.add(weeklyMonthTemperatures[i][j]);
+            }
+        }
+        //Sets my min and max
+        // Source: http://www.java2s.com/Tutorial/Java/0140__Collections/FindMinimumelementofHashSet.htm
+        Object min = Collections.min(temperatures);
+        Object max = Collections.max(temperatures);
+
+
+        //Source: https://stackoverflow.com/questions/14534767/how-to-append-a-newline-to-stringbuilder
+        buildStr.append("High: ").append(max).append(System.getProperty("line.separator"));
+        buildStr.append("Low: ").append(min).append(System.getProperty("line.separator"));
+
+
+        for (int i = (int) min; i < (int) max ; i++) {
+
+            if(!temperatures.contains(i)){
+
+              buildStr.append(outputStr).append(i).append(System.getProperty("line.separator"));
```

Comment:
When you have long lines of code like this you can put each new method on a new line. This is completely up to you though!

`buildStr.append(outputStr)`
     `.append(i)`
     `.append(System.getProperty("line.separator"));`